acmed/config/acmed.toml

```diff
@@ -1,4 +1,10 @@
+# ------------------------------------------------------------------------
+# Base configruation for ACMEd
+# You should adapt this file to include/enhance with your custom toml's
+# ------------------------------------------------------------------------
+
 include = [
     "default_hooks.toml",
     "letsencrypt.toml",
+    "my_server.toml"
```

Comment:
What is this file?

Reply:
Hmm. A typo. It should have been commented out. This should give a user the starting point, to define his account/endpoint definition.
I think a better solution is to provide an example configuration in the wiki / README.md
Given this, just comment it out or delete the line when merging.